src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/format/FormatWorkerAction.kt

```diff
@@ -73,27 +76,19 @@ abstract class FormatWorkerAction : WorkAction<FormatWorkerParameters> {
             },
         )
     }
+}
 
-    private fun formatKt(file: File, ruleSets: Set<RuleProvider>, onError: ErrorHandler) =
-        format(file, ruleSets, onError, false)
+private fun KtLintRuleEngine.formatKt(file: File, onError: ErrorHandler) =
```

Comment:
Moved out of the file to keep function references working after I added a receiver